<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="machine">
      <attribute name="title">
       <string>Станок</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="xBox">
         <property name="title">
          <string>X</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="xLengthLayout">
            <item>
             <widget class="QLabel" name="xLengthLabel">
              <property name="text">
               <string>Длина рабочей области</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="xLengthEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="xScaleLayout">
            <item>
             <widget class="QLabel" name="xScaleLabel">
              <property name="text">
               <string>Масштаб по координате</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="xScaleEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="xVelocityScaleLayout">
            <item>
             <widget class="QLabel" name="xVelocityScaleLabel">
              <property name="text">
               <string>Масштаб по скорости</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="xVelocityScaleEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="xMaxVelocityLayout">
            <item>
             <widget class="QLabel" name="xMaxVelocityLabel">
              <property name="text">
               <string>Максимальная скорость</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="xMaxVelocityEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="yBox">
         <property name="title">
          <string>Y</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="yLengthLayout">
            <item>
             <widget class="QLabel" name="yLengthLabel">
              <property name="text">
               <string>Длина рабочей области</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yLengthEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="yScaleLayout">
            <item>
             <widget class="QLabel" name="yScaleLabel">
              <property name="text">
               <string>Масштаб по координате</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yScaleEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="yVelocityScaleLayout">
            <item>
             <widget class="QLabel" name="yVelocityScaleLabel">
              <property name="text">
               <string>Масштаб по скорости</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yVelocityScaleEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="yMaxVelocityLayout">
            <item>
             <widget class="QLabel" name="yMaxVelocityLabel">
              <property name="text">
               <string>Максимальная скорость</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yMaxVelocityEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="zBox">
         <property name="title">
          <string>Z</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="zLengthLayout">
            <item>
             <widget class="QLabel" name="zLengthLabel">
              <property name="text">
               <string>Блина рабочей области</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="zLengthEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="zScaleLayout">
            <item>
             <widget class="QLabel" name="zScaleLabel">
              <property name="text">
               <string>Масштаб по координате</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="zScaleEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="zVelocityScaleLayout">
            <item>
             <widget class="QLabel" name="zVelocityScaleLabel">
              <property name="text">
               <string>Масштаб по скорости</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="zVelocityScaleEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="zMaxVelocityLayout">
            <item>
             <widget class="QLabel" name="zMaxVelocityLabel">
              <property name="text">
               <string>Максимальная скорость</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="zMaxVelocityEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="fBox">
         <property name="title">
          <string>F</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="fLengthlLayout">
            <item>
             <widget class="QLabel" name="fLengthLabel">
              <property name="text">
               <string>Длина рабочей области</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fLengthEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="fScaleLayout">
            <item>
             <widget class="QLabel" name="fScalelabel">
              <property name="text">
               <string>Масштаб по координате</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fScaleEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="fVelocityScaleLayout">
            <item>
             <widget class="QLabel" name="fVelocityScaleLabel">
              <property name="text">
               <string>Масштаб по скоростям</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fVelocityScaleEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="fMaxVelocityLayout">
            <item>
             <widget class="QLabel" name="fMaxVelocityLabel">
              <property name="text">
               <string>Максимальная скорость</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fMaxVelocityEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="main">
      <attribute name="title">
       <string>Основные</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Размерность координат станка, размеров и отступов для фрез</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mmCheckBox">
         <property name="text">
          <string>Миллиметры</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="smCheckBox">
         <property name="text">
          <string>Сантиметры</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mCheckBox">
         <property name="text">
          <string>Метры</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
